rosa_hcp/rosa-hcp-egress-lockdown-install.adoc

```diff
@@ -9,6 +9,8 @@ Creating a {product-title} (ROSA) cluster with egress lockdown provides a way to
 
 All public and private clusters with egress lockdown get their Red{nbsp}Hat container images from an Amazon Elastic Container Registry (ECR) located in the local region of the cluster instead of gathering these images from various endpoints and registries on the internet. ECR provides storage for OpenShift release images as well as Red{nbsp}Hat Operators. All requests for ECR are kept within your AWS network by serving them over a VPC endpoint within your cluster.
 
+Zero-egress clusters use an Amazon ECR to create a highly secure network architecture that blocks outbound traffic from the cluster to the public internet. Because the necessary cluster communication occurs over private network channels, Amazon ECR serves as the cornerstone by storing and gathering the necessary images. For more on Amazon ECR, see link:https://aws.amazon.com/ecr/[Amazon Elastic Container Registry].
```

Comment:
```suggestion
ROSA Egress Lockdown clusters use AWS ECR to provision ROSA HCP clusters without the need for public internet. Because necessary cluster lifecycle processes occur over AWS private networking, AWS ECR serves as a critical service for core cluster platform images. For more information on AWS ECR, see link:https://aws.amazon.com/ecr/[Amazon Elastic Container Registry].
```